debugging/kernel-logger/kp_kernel_logger.cpp

```diff
@@ -100,7 +100,7 @@ extern "C" void kokkosp_begin_parallel_scan(const char* name,
   printf("    %s\n", name);
 }
 
-extern "C" void kokkospk_end_parallel_scan(const uint64_t kID) {
+extern "C" void kokkosp_end_parallel_scan(const uint64_t kID) {
```

Comment:
Part of a different pull request. Please rebase once that's merged or remove duplicated changes.